### PR TITLE
Add java layer contributor tests

### PR DESCRIPTION
**Pull request by menehune23**
Adds unit tests for Java layer contributors:

- [x] `Function`
- [x] `Invoker`

### CODE REVIEW

File: buildpacks/java/tests/function_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
+// Copyright 2021-2022 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package tests
```
Comment (menehune23) [inline buildpacks/java/tests/function_test.go:4]: Focus here. This is one of two main test suites added.

File: buildpacks/java/tests/invoker_test.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
+// Copyright 2021-2022 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package tests
```
Comment (menehune23) [inline buildpacks/java/tests/invoker_test.go:4]: Focus here too. This is the other new test suite.

File: buildpacks/python/python/build.go [hunk 3]
```diff
@@ -67,10 +67,10 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return result, nil
 	}
 
-	functionClass, isFuncDefDefault := cr.Resolve("BP_FUNCTION")
+	functionClass, isOverride := cr.Resolve("BP_FUNCTION")
```
Comment (menehune23) [inline buildpacks/python/python/build.go:70]: Changes here and below are for consistency with java changes. Kept them minimal.

File: buildpacks/java/java/build.go [hunk 4]
```diff
@@ -49,16 +49,16 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return result, nil
 	}
 
-	functionClass, isFuncDefDefault := cr.Resolve("BP_FUNCTION")
+	functionClass, isOverride := cr.Resolve("BP_FUNCTION")
```
Comment (menehune23) [inline buildpacks/java/java/build.go:52]: This variable was named the opposite of what it should be (thanks @andrew-su for pointing this out)

**Review by andrew-su — APPROVED** [review]
LGTM, thanks!

**Pull request merged**